.gitignore

```diff
@@ -34,3 +34,5 @@ npm-debug.log
 /priv/static/
 
 /.elixir_ls/
+
+/assets/package-lock.json
```

Comment:
The `package-lock.json` file should be committed ([docs](https://docs.npmjs.com/configuring-npm/package-locks.html)).

Reply:
So here are my full thoughts on this:

First of all, the documentation for this repo advises using `npm install` in the `assets` directory, which will overwrite the lockfile.

![image](https://user-images.githubusercontent.com/31600776/80162887-26c31b80-859a-11ea-8a77-54ea670c2ec4.png)


Additionally, we are not distributing a package which would benefit from having locked dependencies, but I think that excluding the lockfile in that case is more a matter of taste.

I don't see a reason to exclude the lockfile from source control other than to clean up diffs, and that's not a compelling enough point for me to want to chase this point down a rabbit hole, so I'll revert the change. How would you feel about replacing `npm install` with `npm install --no-save` in the docs?

Reply:
https://github.com/the-pat/iaas/pull/2 @the-pat